### DISCUSSION TITLE
ongoing development

### DISCUSSION THREAD

**Discussion by sjgoalie** [debauchee/barrier#1712] [General]
I dont want to sound like one of those this-project-is-dead-blah blah types, but is it?  It's been months since there was a checkin, even before that, releases are a year+ appart.   I use this extensively but if it's dying I need to look for an alternative.

**Comment by jjohnson42** [comment]
From what I see:
https://github.com/input-leap/input-leap

Basically looks like the people doing work on barrier anymore went to that project because they couldn't have access to barrier:
https://github.com/input-leap/input-leap/issues/1414

**Reply by hannesrudolph** [reply]
This one is a pain to install. I wonder if there are others around.

**Reply by sjgoalie** [reply]
From what I can tell, they are all a hot mess.   synergy is poorly maintained.   Barrier is worse.   Input-leap doesnt seem to know how to get started even though its been around for a while now, they have 600+ open issues, don't tag releases (last one in 2021), don't build binaries so no one can test without a build environment.  I feel like there is a huge hole here that someone with some talent and some $$ could jump in and take it all.

**Comment by philter1980** [comment]
I would like to suggest to add a new feature: set the same desktop image on all connected clients to have an uniform layer
